charts/merlin/values.yaml

```diff
@@ -23,6 +23,9 @@ merlin:
   apiHost: http://merlin.mlp/v1
   docsURL: ""
 
+  # Comma-separated value of Docker registries that can be chosen in deployment page
+  dockerRegistries: ghcr.io/gojek,ghcr.io/your-company
```

Comment:
I am not sure about this feature. I am thinking we should just ask user the fully qualified docker image name in the UI instead of have a suggestion as combo box. 
It's one less feature and configuration to maintain.

Reply:
This feature makes sure the user only uses the supported Docker registry by Merlin. It prevents the failure of Image pulling if the model cluster doesn't have the credential to access it. Turing also has this feature and by having this in Merlin we can provide the same experience across MLP. If the user wants to use another Docker registry, he/she needs to ask the administrator to add it to Merlin and to make sure the model cluster can access it (by adding Image Pull Secret). For default, we have public Docker Hub + dockerRegistries specified here.

Reply:
Ok got it